src/components/overview/Overview.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import { EmbeddedResource, GeoJSONResource, JSONResource } from '@/schemas/configuration-schema';
+
+import React from 'react';
+// eslint-disable-next-line import/named
+import { TableColumn } from 'react-data-table-component';
+import { createSharedPathnamesNavigation } from 'next-intl/navigation';
+import dynamic from 'next/dynamic';
+import { getColorForResourceType } from '@/colors';
+import { locales } from '@/locales';
+import { replaceWhitespaceInString } from '@/utils/stringutils';
+import { useTranslations } from 'next-intl';
+
+// Avoid hydration error inside Table pagination https://stackoverflow.com/questions/77763766/next-js-hydration-error-with-shadcn-dialog-component
+const DataTable = dynamic(() => import('react-data-table-component'), { ssr: false });
+
+const { Link } = createSharedPathnamesNavigation({
+  locales: [...locales.values()],
+});
+
+// TODO: We need a way to select the subcategory: Either in the navigation, or inside of a selector component at the top of the overview
+export default function Overview({
+  resources,
+  categories,
+}: {
+  resources: (EmbeddedResource | GeoJSONResource | JSONResource)[];
+  categories: string[] | undefined;
+}) {
+  const tableT = useTranslations('Table');
+  const columns = [
+    {
+      cell: (row: { html: React.JSX.Element }) => row.html,
+    },
+  ] as TableColumn<unknown>[];
+  return (
+    <div className="flex-fill">
+      <DataTable
+        columns={columns}
+        data={getDataForResources(resources, categories)}
+        noDataComponent={tableT('noRecords')}
+        highlightOnHover
+        pagination
+        paginationComponentOptions={{
+          rowsPerPageText: tableT('rowsPerPageText'),
+          rangeSeparatorText: tableT('rangeSeparatorText'),
+          selectAllRowsItem: true,
+          selectAllRowsItemText: tableT('selectAllRowsItemText'),
+        }}
+      />
+    </div>
+  );
+}
+
+function getDataForResources(
```

Comment:
We could consider moving this mapping out to the `page.tsx`. See my thought in the "long" comment